examples/language-modeling/run_lora_clm.py

```diff
@@ -263,6 +263,10 @@ class DataArguments:
         default=False,
         metadata={"help": "Whether to have a SQL style prompt"},
     )
+    chat_prompt: bool = field(
+        default=False,
+        metadata={"help": "Whether to have a chat style prompt"},
```

Comment:
```suggestion
        metadata={"help": "Whether to have a chat style prompt."},
```

Please add the period for sql_prompt metadata as well.